bliss/surveys/dc2.py

```diff
@@ -18,7 +18,7 @@ class DC2(Survey):
     # why are these bands out of order? why does a test break if they are ordered correctly?
     BANDS = ("g", "i", "r", "u", "y", "z")
 
-    def __init__(self, data_dir, cat_path, batch_size, n_split, image_lim):
+    def __init__(self, data_dir, cat_path, batch_size, n_split, image_lim, for_plotting=False):
```

Comment:
It's cleaner if the DC2 survey class doesn't "know" about whether it's being used for plotting or not. Survey classes are just supposed to provide access to the underlying data. Better to move the logic elsewhere, presumably to where the DC2 dataset is being used.